src/x11/x.c

```diff
@@ -69,6 +69,48 @@ static void setopacity(Window win, unsigned long opacity);
 static void x_handle_click(XEvent ev);
 static void x_win_setup(void);
 
+/* see x.h */
+const char *xevent_type_to_string(int type)
```

Comment:
I'm not sure about having this monster of a function just to log a debug message :/

Reply:
We definitely can remove it. It was just super useful while debugging and I left it in on purpose. I wanted to know, what you think about it.